src/lib/y2network/autoinst_profile/interface_section.rb

```diff
@@ -0,0 +1,320 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/autoinst_profile/section_with_attributes"
+
+module Y2Network
+  module AutoinstProfile
+    # This class represents an AutoYaST <interface> section under <interfaces>
+    #
+    #  <interface>
+    #    <bootproto>static</bootproto>
+    #    <broadcast>127.255.255.255</broadcast>
+    #    <device>lo</device>
+    #    <firewall>no</firewall>
+    #    <ipaddr>127.0.0.1</ipaddr>
+    #    <netmask>255.0.0.0</netmask>
+    #    <network>127.0.0.0</network>
+    #    <prefixlen>8</prefixlen>
+    #    <startmode>nfsroot</startmode>
+    #    <usercontrol>no</usercontrol>
+    #  </interface>
+    #
+    # @see InterfacesSection
+    class InterfaceSection < SectionWithAttributes
+      def self.attributes
+        [
+          { name: :bootproto },
+          { name: :broadcast },
+          { name: :device },
+          { name: :name }, # has precedence over device
+          { name: :ipaddr },
+          { name: :remote_ipaddr },
+          { name: :netmask },
+          { name: :network }, # TODO: what it is? looks like ipaddr with applied prefix
```

Comment:
it can be "network address" - exactly what is in the comment. However I've never seen it in any profile, so what about trying to remove it silently ?

Reply:
Well, my plan is to ignore it for now, but keep it in rnc for backward compatibility. That also reminds me that we should improve official SLE documentation for autoyast network as current one is really insuficient.